code/game/objects/item.dm

```diff
@@ -350,6 +359,30 @@
 /obj/item/proc/moved(mob/user, old_loc)
 	return
 
+/obj/item/proc/get_volume_by_throwforce_and_or_w_class()
+	if(throwforce && w_class)
+		return Clamp((throwforce + w_class) * 5, 30, 100)// Add the item's throwforce to its weight class and multiply by 5, then clamp the value between 30 and 100
+	else if(w_class)
+		return Clamp(w_class * 8, 20, 100) // Multiply the item's weight class by 8, then clamp the value between 20 and 100
+	else
+		return 0
+
+/obj/item/throw_impact(atom/hit_atom)
+	..()
+	if(isliving(hit_atom)) //Living mobs handle hit sounds differently.
```

Comment:
Maybe this block should be on `/mob` or even `/atom`? As Psi said, it's a bit odd for objects to define their own mob hit sound, rather than the mobs themselves.